exporter/zipkinexporter/README.md

```diff
@@ -1,10 +1,14 @@
 # Zipkin Exporter
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | trace     |
+| Distributions            | [contrib] |
```

Comment:
This is also [available in core](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol/manifest.yaml#L32)

Reply:
ok